kube_proxy/pyproject.toml

```diff
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.11.2",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-kube-proxy"
+description = "The Kube Proxy check"
+readme = "README.md"
+license = "BSD-3-Clause"
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "kube_proxy",
```

Comment:
```suggestion
    "kube_proxy",
    "kubernetes",
```

Reply:
These in fact do nothing because we don't upload to PyPI (which uses them for search)